lib/axiom/fs/stream/writable_stream.js

```diff
@@ -12,15 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import AxiomEvent from 'axiom/core/event';
+
 /** @typedef function():void */
-var WriteCallback;
+export var WriteCallback;
```

Comment:
I thought it was needed because it was used in a Closure comment in `writable_stream_forwarder.js`, but just checked, and the error I was getting was due to something else. Reverted.